qa/workunits/rbd/cli_migration.sh

```diff
@@ -39,6 +41,23 @@ create_base_image() {
     rbd bench --io-type write --io-pattern rand --io-size=4K --io-total 128M ${image}
 }
 
+create_base_image_with_namespace() {
```

Comment:
This function is a copy of `create_base_image()` with the exception of `rbd namespace create` command.  I'd suggest creating the namespace at the top level and passing it to the existing `create_base_image()` as part of the image spec:
```
rbd namespace create rbd/${NAMESPACE1}
create_base_image rbd/${NAMESPACE1}/${IMAGE1}
test_import_native_format_with_namespace ...
```